src/windows/TextInput.tsx

```diff
@@ -17,11 +17,17 @@ import { TextInput as TextInputBase } from '../native-common/TextInput';
 export class TextInput extends TextInputBase implements FocusManagerFocusableComponent {
 
     protected _render(props: RN.TextInputProps): JSX.Element {
+        // Work around the fact that the react-native type declaration file doesn't
+        // currently define tabIndex and incorrectly defines the onFocus prop.
+        const undefinedProps: any = {
+            tabIndex: this.getTabIndex(),
+            onFocus: (e: React.FocusEvent<any>) => this._onFocusEx(e, props.onFocus)
```

Comment:
>onFocus: (e: React.FocusEvent<any>) => this._onFocusEx(e, props.onFocus) [](start = 12, length = 72)

This line gives me an "npm run build" error when run in a standalone enlistment:

Argument of type '((e: NativeSyntheticEvent<TextInputFocusEventData>) => void) | undefined' is not assignable to parameter of type '((e: FocusEvent<any>) => void) | undefined'.
  Type '(e: NativeSyntheticEvent<TextInputFocusEventData>) => void' is not assignable to type '((e: FocusEvent<any>) => void) | undefined'.
    Type '(e: NativeSyntheticEvent<TextInputFocusEventData>) => void' is not assignable to type '(e: FocusEvent<any>) => void'.
      Types of parameters 'e' and 'e' are incompatible.
        Type 'FocusEvent<any>' is not assignable to type 'NativeSyntheticEvent<TextInputFocusEventData>'.
          Types of property 'nativeEvent' are incompatible.
            Type 'FocusEvent' is not assignable to type 'TextInputFocusEventData'.
              Property 'text' is missing in type 'FocusEvent'.

Reply:
Not sure why this didn't show up before. It's fixed now.